src/main/resources/application.properties

```diff
@@ -13,13 +13,13 @@ spring.web.resources.cache.cachecontrol.must-revalidate=true
 spring.web.resources.static-locations=classpath:/api/
 db.migrate.dropAllOnStart=false
 db.migrate.updateAllOnStart=true
-db.datarepo.uri=jdbc:postgresql://127.0.0.1:5432/datarepo
+db.datarepo.uri=jdbc:postgresql://${DATABASE_NAME:127.0.0.1}:5432/datarepo
```

Comment:
Why is `DATABASE_NAME` needed here? Is that due to running inside docker? If so, does this still work when running outside docker?

Reply:
Due to running inside docker. The second bit is a default, so if DATABASE_NAME isn't set, it defaults to the same value